### PR TITLE
Remove unused codes in NewAPIServerCommand

### PR DESCRIPTION
**Pull request by gouyang**
Signed-off-by: Guohua Ouyang gouyang@redhat.com

### CODE REVIEW

File: cmd/kube-apiserver/app/server.go [hunk 1]
```diff
@@ -151,8 +151,6 @@ func NewAPIServer() *APIServer {
 
 // NewAPIServerCommand creates a *cobra.Command object with default parameters
 func NewAPIServerCommand() *cobra.Command {
-	s := NewAPIServer()
-	s.AddFlags(pflag.CommandLine)
```
Comment (mikedanese) [inline cmd/kube-apiserver/app/server.go]: I think it has the effect of binding all the server flags to the command line. This has side effects.

**Comment by mikedanese** [comment]
cc @feiskyer

File: cmd/kube-apiserver/app/server.go (hunk 1 above)
Comment (gouyang) [inline cmd/kube-apiserver/app/server.go]: I tested it with genkubedocs, seems no side effects.

**Comment by ashcrow** [comment]
Travis error:

```Generated docs are out of date. Please run hack/update-generated-docs.sh```

**Comment by gouyang** [comment]
Close this as it failed to generate complete doc contents.

**Pull request closed without merging**